pkg/dequeuer/dequeuer_test.go

```diff
@@ -351,7 +351,7 @@ func TestSQSDequeuer_Start_HandlerError(t *testing.T) {
 
 	dq.waitTimeSeconds = aws.Int64(0)
 	dq.notFoundSleepTime = 0
-	dq.renewalPeriod = time.Second
+	dq.renewalPeriod = 500 * time.Millisecond
```

Comment:
Why the 500ms renewal? And should we do this the same for the one in `TestSQSDequeuer_StartMessageRenewer`?

Reply:
it doesn't matter that much, I just wanted to make sure it was fast enough

Reply:
Cool!